src/main/java/gregtech/common/items/behaviors/filter/BaseFilterUIManager.java

```diff
@@ -0,0 +1,53 @@
+package gregtech.common.items.behaviors.filter;
+
+import gregtech.api.cover.CoverWithUI;
+import gregtech.api.items.gui.ItemUIFactory;
+import gregtech.api.items.metaitem.stats.IItemBehaviour;
+import gregtech.api.mui.GTGuiTheme;
+import gregtech.api.mui.GTGuis;
+import gregtech.api.mui.factory.MetaItemGuiFactory;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.world.World;
+
+import com.cleanroommc.modularui.api.drawable.IKey;
+import com.cleanroommc.modularui.api.widget.Interactable;
+import com.cleanroommc.modularui.factory.HandGuiData;
+import com.cleanroommc.modularui.screen.ModularPanel;
+import com.cleanroommc.modularui.utils.Alignment;
+import com.cleanroommc.modularui.value.sync.GuiSyncManager;
+
+public abstract class BaseFilterUIManager implements IItemBehaviour, ItemUIFactory {
+
+    protected static final ModularPanel ERROR = GTGuis.createPanel("ERROR", 100, 100)
+            .child(IKey.str("This is an error!").asWidget().align(Alignment.Center));
```

Comment:
Should this be `lang` instead of `str` so it can be localized? Also, this seems unused.

Reply:
this was originally used for when a filter was null, but since it can't be null now this can be removed.